sponnet/pipeline.libsonnet

```diff
@@ -173,6 +173,11 @@
       payloadConstraints: {},
       addPayloadConstraints(key, value):: self + { payloadConstraints+: { [key]: value } },
     },
+    pipeline(name, application, pipeline, statuses):: trigger(name, 'pipeline') {
```

Comment:
Can the keys for `type: "pipeline"` and `enabled: true | false` be added?

Reply:
They are already added by the `trigger()` function: https://github.com/spinnaker/spinnaker/blob/8757a9d151f8daaaf4b91c890b6faa7cf3817420/sponnet/pipeline.libsonnet#L136-L139

Should I override them?

Reply:
oof, I missed that. Thanks!